### ISSUE TITLE
App crashes when adding a user to chat? (Fixed - See solution)

### ISSUE THREAD

**Issue by SimplisticApps** [chat-sdk/chat-sdk-android#48]
Okay never mind, i know what was wrong!  it looks like the ID of AUDIO_PICKER_ID was the same as ADD_USERS so it was trying to upload a file without being told!

So this is should be changed in the SDK to stop it happening again

I changed id of line: 78 from 103 to 120
Thanks

**Comment by SimplisticApps** [comment]
Oop's sorry about that, like an idiot i tried multiple times to add the bug label lol!

**Comment by kykrueger** [comment]
Hi, 

I see this is marked as fixed, but I have implemented a different solution in the main repo. 
ADD_USERS and SHOW_DETAILS have been moved from `ChatSDKAbstractChatActivity` to the `ChatSDKChatHelper` class where the other ints are declared. This should prevent reuse of the values in the future. It wasn't smart of us to have had variables declared in both classes before.

While an enum should be even better, I remember there being a reason why one was not used in the first place. 

-- Kyle
-- Kyle
